### PR TITLE
add some cisco device types of different series

### PR DESCRIPTION
**Pull request by abu-rehan**
Hi! I used existing schemas as reference. I believe I have put in all the mandatory info. Some optional info may be missing but would still like this PR to get accepted if possible... :-)

### CODE REVIEW

File: device-types/Cisco/C9300L-24UXG-4X.yaml [hunk 1]
```diff
@@ -0,0 +1,132 @@
+---
+manufacturer: Cisco
+model: Catalyst 9300L-24UXG-4X
+part_number: C9300L-24UXG-4X
+slug: cisco-c9300l-24uxg-4x
+u_height: 1
+is_full_depth: true
```
Comment (danner26) [inline device-types/Cisco/C9300L-24UXG-4X.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Cisco/WS-C3550-48-SMI.yaml [hunk 2]
```diff
@@ -0,0 +1,113 @@
+---
+manufacturer: Cisco
+model: WS-C3550-48-SMI
+slug: cisco-ws-c3550-48-smi
+part_number: WS-C3550-48-SMI
+u_height: 1
```
Comment (danner26) [inline device-types/Cisco/WS-C3550-48-SMI.yaml:6]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Cisco/WS-C3560G-24TS-E.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
+---
+manufacturer: Cisco
+model: Catalyst 3560G-24TS-E
+slug: cisco-ws-c3560g-24ts-e
+part_number: WS-C3560G-24TS-E
+is_full_depth: false
+u_height: 1
```
Comment (danner26) [inline device-types/Cisco/WS-C3560G-24TS-E.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Cisco/WS-C3560G-24TS-S.yaml [hunk 4]
```diff
@@ -0,0 +1,74 @@
+---
+manufacturer: Cisco
+model: Catalyst 3560G-24TS-S
+slug: cisco-ws-c3560g-24ts-s
+part_number: WS-C3560G-24TS-S
+is_full_depth: false
+u_height: 1
```
Comment (danner26) [inline device-types/Cisco/WS-C3560G-24TS-S.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Cisco/WS-C3560G-48PS-E.yaml [hunk 5]
```diff
@@ -0,0 +1,218 @@
+---
+manufacturer: Cisco
+model: Catalyst 3560G-48PS-E
+slug: cisco-ws-c3560g-48ps-e
+part_number: WS-C3560G-48PS-E
+is_full_depth: false
+u_height: 1
```
Comment (danner26) [inline device-types/Cisco/WS-C3560G-48PS-E.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

**Review by danner26 — CHANGES_REQUESTED** [review]

**Review by DanSheps — APPROVED** [review]
These are all Catalyst switches so the weight and airflow is easy to find on the datasheet.  Please add them and we can get this merged in.

**Pull request closed without merging**